### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Updates deps

* Closes #1040 
* Closes #1039 
* Closes #1038 
* Closes #1037 
* Closes #1036 
* Closes #1035

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,10 +65,10 @@
     "visual-testing-app:serve": "serve visual-testing-app/dist --no-clipboard --single --listen 3001"
   },
   "dependencies": {
-    "@emotion/core": "10.0.16",
-    "@emotion/styled": "10.0.15",
+    "@emotion/core": "10.0.17",
+    "@emotion/styled": "10.0.17",
     "common-tags": "1.8.0",
-    "dom-helpers": "3.4.0",
+    "dom-helpers": "5.0.1",
```
Comment (montezume) [inline package.json]: drops IE 8 support ⚱️

Comment (tdeekens) [inline package.json]: Oh wow. What changed here?

Comment (montezume) [reply package.json]: https://github.com/react-bootstrap/dom-helpers/commit/87e2fe448b5e852574e1ee63bd3f2102b4c65d79

Looks like they dropped IE 8 support and "removed a lot of code"

Comment (montezume) [reply package.json]: okay it looks like it did more than that 🤔

Comment (montezume) [reply package.json]: looks like it just moved the paths around. the code is exactly the same

**Pull request merged**